frontend/validators.js

```diff
@@ -1,4 +1,6 @@
 const validateMnemonic = require('../wallet/mnemonic').validateMnemonic
```

Comment:
I would merge this three lines into one statement:

const {a, b, c } = require('x')